client/my-sites/domains/email-providers-upsell/index.jsx

```diff
@@ -41,7 +42,14 @@ export default function EmailProvidersUpsell( { domain } ) {
 					skipButtonLabel={ translate( 'Skip' ) }
 					titanProductSlug={ TITAN_MAIL_YEARLY_SLUG }
 				/>
-			) : null }
+			) : (
+				<EmailProvidersStackedComparison
+					cartDomainName={ domain }
+					comparisonContext="domain-upsell"
+					selectedDomainName={ domain }
```

Comment:
We pass the same domain name in `cartDomainName` and `selectedDomainName`. I think we could simplify this a bit, and makes it less confusing (e.g. by passing a boolean such as `isDomainInCart` instead).